### PR TITLE
adding new rules for user account manipulation, parsing HTTP URLs, and registering raw devices

### PR DESCRIPTION
**Pull request by mike-hunhoff**

### CODE REVIEW

File: nursery/parse-http-url.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# generated using capa explorer for IDA Pro
+rule:
+  meta:
+    name: parse HTTP URL
```
Comment (mr-tz) [inline nursery/parse-http-url.yml:4]:
```suggestion
    name: parse URL
```

**Review by mr-tz — APPROVED** [review]
looks good, optionally add MITRE ATT&CK: https://attack.mitre.org/techniques/T1136/, https://attack.mitre.org/techniques/T1087/, https://attack.mitre.org/techniques/T1098/

File: nursery/list-user-account-groups.yml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# generated using capa explorer for IDA Pro
+rule:
+  meta:
+    name: list user account groups
+    namespace: host-interaction/accounts
```
Comment (williballenthin) [inline nursery/list-user-account-groups.yml:5]: can you add a description here that differentiates this rule from `list groups for user account`?

Comment (williballenthin) [reply nursery/list-user-account-groups.yml:5]:
```suggestion
    namespace: host-interaction/accounts
    description: enumerates all the groups present on the system/domain
```

maybe?

File: nursery/list-groups-for-user-account.yml [hunk 3]
```diff
@@ -0,0 +1,11 @@
+# generated using capa explorer for IDA Pro
+rule:
+  meta:
+    name: list groups for user account
```
Comment (williballenthin) [inline nursery/list-groups-for-user-account.yml:4]:
```suggestion
    name: list groups for user account
    description: enumerates all the groups to which a user account belongs
```
maybe?

**Review by williballenthin — APPROVED** [review]

File: nursery/list-user-account-groups.yml (hunk 2 above)
Comment (mike-hunhoff) [reply nursery/list-user-account-groups.yml:5]: no problem I can see these names getting confusing. love the suggested description.

File: nursery/list-groups-for-user-account.yml (hunk 3 above)
Comment (mike-hunhoff) [reply nursery/list-groups-for-user-account.yml:4]: see above - great suggestion

**Pull request merged**